### DISCUSSION TITLE
Front End Stack - Quais caminhos posso seguir

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#75] [Discussão]
Todos nós sabemos que no mundo do front existem diversas ferramentas, tecnologias, metodologias etc.
Gostaria de trazer a discussão de quais caminhos podemos seguir no meio profissional, qual "combo" de tecnologias podemos aprender e quais estão bombando no mercado

**Comment by afonsopacifer** [comment]
Opa,
Eu cheguei a apresentar em algumas faculdades aqui no Rio a talk **Fron-End no mundo real**,
com o objetivo de tentar levar um pouco da nossa realidade para a galera que ainda não trampa e só tem a facul como referência..

Ai os Slides, talvez ajude :)

https://speakerdeck.com/afonsopacifer/fron-end-no-mundo-real

Esse lance de Stack é complicado, já que caso é um caso e essas coisas, mas se vc focar nos conceitos e nos **padrões** você vai ficar bem ;)

**Comment by Cleysonlb** [comment]
Boa @afonsopacifer !!!!!!!!!! :D

**Comment by felipefialho** [comment]
Fala @Cleysonlb, na issue #54, o pessoal mostra um pouco do workflow e as ferramentas que estão utilizando.

Como bem disse o @afonsopacifer, Workflow é uma coisa muito pessoal e pode inclusive variar muito dependendo do projeto.

Mas recomendo você criar pelo menos um coding-style pra você seguir, tipo [esse meu](https://github.com/LFeh/coding-style)(desatualizado). Já ajuda muito e os padrões em tese, deveriam ser usados sempre.

**Comment by Cleysonlb** [comment]
Boa @LFeh  curti esse coding-style, vou implementar no meu workflow!!
